sample_xblocks/filethumbs/filethumbs.py

```diff
@@ -59,9 +64,9 @@ def student_view(self, context=None):  # pylint: disable=W0613
         frag = Fragment(unicode(html_str))
 
         if not self.fs.exists("thumbsvotes.json"):
-            with self.fs.open("thumbsvotes.json", "wb") as f:
-                json.dump({'up': 0, 'down': 0}, f)
-                f.close()
+            with self.fs.open('thumbsvotes.json', 'wb') as file_output:
+                json.dump({'up': 0, 'down': 0}, file_output)
+                file_output.close()
```

Comment:
I'm fine with changing `f` to `file_output`, but personally, I'm also OK with just marking `f` as an acceptable variable name in the `pylintrc` file. I think that `f` is a clear name in context, especially since it's widely used for this purpose, and the shortness of the name indicates that the variable is short-lived.